scenario_result/README.md

```diff
@@ -0,0 +1 @@
+Directory for screenshotes with failed test scenarios
```

Comment:
It would be awesome to add this dir to .gitignore to prevent accidental commits of local screenshots.
Having readme inside, explaining directory purpose, may also be desired.